spec/index.bs

```diff
@@ -13714,6 +13726,52 @@ In WebGPU, it does so by [$Replace the drawing buffer|replacing the drawing buff
     validation will fail inside {{GPUCanvasContext/getCurrentTexture()}}.
 </div>
 
+<h3 id=gpucanvascolormode data-dfn-type=enum>`GPUCanvasColorMode`
+<span id=GPUCanvasColorMode></span>
+</h3>
+
+This enum specifies how color values are displayed to the screen.
+
+<dl dfn-type=enum-value dfn-for=GPUCanvasColorMode>
+    : <dfn>"default"</dfn>
+    ::
+        Clamp color values to the `[0, 1]` interval in the color space of the screen.
+
+        <div class=example>
+            For example, suppose that the value `(1.035, -0.175, -0.140)` is written to an
+            `'srgb'` canvas.
+
+            If this is presented to an sRGB screen, then this will be converted to sRGB
+            (which is a no-op, because the canvas is sRGB), and then will be clamped to
+            the sRGB value `(1.0, 0.0, 0.0)`.
+
+            If this is presented to a Display P3 screen, then this will be converted to
+            the value `(0.948, 0.106, 0.01)` in the Display P3 color space, and no
+            clamping will be needed.
+        </div>
+
+    : <dfn>"extended"</dfn>
+    ::
+        Clamp color values only to the capabilities of the screen, allowing values
+        outside of the `[0, 1]` interval.
+
+        <div class=example>
+            For example, suppose that the value `(2.5, -0.15, -0.15)` is written to an
+            `'srgb'` canvas.
+
+            If this is presented to an sRGB screen that is capable of displaying values
+            in the `[0,4]` interval in sRGB space, then this will be converted to sRGB
+            (which is a no-op, because the canvas is sRGB), and then will be clamped to
+            the sRGB value `(2.5, 0.0, 0.0)`.
+
+            If this is presented to a Display P3 screen that is capable of displaying
+            values in the `[0,2]` interval in Display P3 space, then this will be
```

Comment:
Would it ever be the case that the interval is different for each RGB channel? If so the worked examples could say something like "[0, 2] in each of the R/G/B channels"

Reply:
I think it's a safe assumption that the max is the same for all channels. I've never seen anything where R/G/B have different ranges, and it would be a strange world if such a thing were to happen (because then R=G=B=max wouldn't be white -- it would have some color).

Even with things like f.lux, where white isn't white (but rather is yellow or so), that "max brightness" is still R=G=B=max in terms of pixel value.

Reply:
Good point, thanks!